libraries/botbuilder-core/src/turnContext.ts

```diff
@@ -764,14 +765,25 @@ export class TurnContext {
      * Gets the locale stored in the turnState.
      */
     public get locale(): string {
-        return this._turnState.get(this._turnLocale);
+        const turnObj = this._turnState.get(this._turn);
+        const locale = turnObj.get(this._locale);
+        if (locale && typeof locale === 'string') {
```

Comment:
You just need `typeof locale === 'string'`.

```bash
$ node
> typeof null === 'string'
false
> typeof undefined === 'string'
false
```

Reply:
changed